src/logic/calulate.test.js

```diff
@@ -0,0 +1,11 @@
+import calculate from './calculate';
+
+describe('Button 0', () => {
+  test('display 0 when button zero is pressed', () => {
+    const op = { next: '0' };
+    const expected = {};
+    const buttonName = '0';
+    const result = calculate(op, buttonName);
+    expect(result).toEqual(expected);
+  });
+});
```

Comment:
- [ ]  I would suggest you please add more tests in your calculator.test.js as all the functionalities and scenarios should be tested.